app/_hub/kong-inc/rate-limiting-advanced/index.md

```diff
@@ -2,7 +2,7 @@
 
 name: Rate Limiting Advanced
 publisher: Kong Inc.
-version: 1.3-x
+version: 0.37-x
```

Comment:
```suggestion
version: 1.3-x
```

Reply:
whoops didn't mean to commit that change. Yes this fixes the dropdown bug.